### ISSUE TITLE
helm pull errors are not logged

### ISSUE THREAD

**Issue by msteinhoff** [holos-run/holos#332]
Holos Version: 0.98.1

When I add a helm component with a version that does not exist, like this:

```
package holos

// Produce a helm chart build plan.
holos: HelmChart.BuildPlan

HelmChart: #Helm & {
	Name: "podinfo"
	Chart: {
		version: "0.0.0"
		repository: {
			name: "podinfo"
			url:  "https://stefanprodan.github.io/podinfo"
		}
	}
}
```

holos fails (as expected) but doesnt tell me where it failed:

```
> holos render platform ./platform

could not run: could not run helm pull: exit status 1 at builder/v1alpha5/builder.go:517
could not run: could not render component: exit status 1 at builder/v1alpha5/builder.go:136
```

increasing the log level to debug doesnt add much info either (but imho this is something that should not require debug logging anyways):

```
> holos render platform ./platform --log-level debug

finalized config from flags
cue: discriminating platform instance
discriminated v1alpha5 Platform
render component
running: holos
could not run: could not run helm pull: exit status 1 at builder/v1alpha5/builder.go:517
could not run: could not render component: exit status 1 at builder/v1alpha5/builder.go:136

**Comment by jeffmccune** [comment]
`holos` should log the helm error to stderr for clear visibility.  This is a bug because it doesn't.

**Comment by jeffmccune** [comment]
Thanks @msteinhoff I'm going to focus this issue on `helm pull`, if you run into other places where errors aren't sufficiently logged please report them as separate issues.

**Comment by jeffmccune** [comment]
@msteinhoff FYI fixed in v0.99.1 thanks again for the report.
